### PR TITLE
Game content

### PR DESCRIPTION
**Pull request by Veradictus**
This update contains many new features, these include backend fixes and improvements, experimental frontend features such as WebGL, but primarily it offers a lot of new in-game content. There are 8 new skills, many new areas to explore, new achievements/quests/items, etc. The full change-log is as follows:

- Added 8 new skills
- Added new quests/achievements/areas/mobs/bosses
- Added new arrows
- Added a crafting system that is used for new skills
- Added hovering silhouette and hurt sprites
- Added map parsing tile property for obstructing tiles
- Increased container size for inventory and bank
- Reorganized the UI such that buttons do not affect mobile experience
- Fixed leaderboards not updating
- Fixes a bug where the death animation would hang
- Fixes a minor bug where the idle animation would interrupt the walking animation
- Fixes bug where status effects would not clear properly
- Fixes relating to new characters and guests not being properly recorded
- Fixes bug with enchanted weapon stacking when placed in the bank
- Fixes bug with enchantments disappearing when dropping item
- Fixes bug with guild status displaying as offline when creating guild
- Preliminary backend security against DoS attacks
- Many backend optimizations, refactoring, and dependency upgrades

### CODE REVIEW

File: packages/client/data/sprites.json [hunk 1]
```diff
@@ -8322,6 +8323,7 @@
   { "id": "items/bluewizardrobe", "offsetX": 0, "offsetY": 0 },
   { "id": "items/book", "offsetX": 0, "offsetY": 0 },
   { "id": "items/boostpotion", "offsetX": 0, "offsetY": 0 },
+  { "id": "items/borth", "offsetX": 0, "offsetY": 0 },
```
Comment (octoshrimpy) [inline packages/client/data/sprites.json]: what is `borth`? is it broth?

File: packages/server/data/items.json [hunk 2]
```diff
@@ -5226,6 +5226,286 @@
     "healAmount": 850
   },
 
+  "barley": {
+    "type": "object",
+    "name": "Barley",
+    "description": "It's a fruit, no it's a sald, no it's a grain. You can probably use it to make bread or something like that.",
```
Comment (octoshrimpy) [inline packages/server/data/items.json:5232]: what is `sald`?

Barley can be used for flour and beer

File: packages/server/data/items.json [hunk 3]
```diff
@@ -5226,6 +5226,286 @@
...
+    "type": "object",
+    "name": "Tin Ore",
+    "description": "A piece of tin ore.",
+    "price": 100
+  },
+
+  "tofu": {
+    "type": "object",
+    "name": "Tofu",
+    "description": "A block of tofu. I don't know what to do with this, I'm not vegan.",
```
Comment (octoshrimpy) [inline packages/server/data/items.json:5491]: tofu can be eaten as is, or cooked with other things.it absorbs the flavors of anything you cook it with, so cooking it with various broths can amplify its default bland flavor

File: packages/server/data/items.json [hunk 4]
```diff
@@ -5226,6 +5226,286 @@
...
+    "type": "object",
+    "name": "Tomato",
+    "description": "A tomato, it's a fruit and it's edible. Yes it's a fruit get over it.",
+    "price": 100
+  },
+
+  "waterbucket": {
+    "type": "object",
+    "name": "Water Bucket",
+    "description": "A bucket of water. It's pretty tasty when cooked.",
```
Comment (octoshrimpy) [inline packages/server/data/items.json:5505]: ....cooked? :joy:

File: packages/client/data/sprites.json (hunk 1 above)
Comment (Pixivic) [reply packages/client/data/sprites.json]: > what is `borth`? is it broth?

Yes, Broth

**Review by lemueldls — APPROVED** [review]

**Pull request merged**